Patch by upstream allows to use a certificate chain files instead of a single
certificate files.

Index: ejabberd-1.1.4/src/tls/tls_drv.c
===================================================================
--- ejabberd-1.1.4.orig/src/tls/tls_drv.c	2008-02-20 13:28:24.000000000 +0300
+++ ejabberd-1.1.4/src/tls/tls_drv.c	2008-02-20 13:28:53.000000000 +0300
@@ -108,8 +108,8 @@
 	 d->ctx = SSL_CTX_new(SSLv23_method());
 	 die_unless(d->ctx, "SSL_CTX_new failed");
 
-	 res = SSL_CTX_use_certificate_file(d->ctx, buf, SSL_FILETYPE_PEM);
-	 die_unless(res > 0, "SSL_CTX_use_certificate_file failed");
+	 res = SSL_CTX_use_certificate_chain_file(d->ctx, buf);
+	 die_unless(res > 0, "SSL_CTX_use_certificate_chain_file failed");
 
 	 res = SSL_CTX_use_PrivateKey_file(d->ctx, buf, SSL_FILETYPE_PEM);
 	 die_unless(res > 0, "SSL_CTX_use_PrivateKey_file failed");
